admin/class-sitemaps-admin.php

```diff
@@ -98,4 +111,71 @@ function status_transition( $new_status, $old_status, $post ) {
 			}
 		}
 	}
+
+	/**
+	 * While bulk importing, just save unique post_types
+	 *
+	 * When importing is done, if we have a post_type that is saved in the sitemap
+	 * try to ping the search engines
+	 *
+	 * @param string   $new_status New post status.
+	 * @param string   $old_status Old post status.
+	 * @param \WP_Post $post       Post object.
+	 */
+	private function status_transition_bulk( $new_status, $old_status, $post ) {
+		if ( ! defined( 'WP_IMPORTING' ) ) {
+			return;
+		}
+
+		if ( $new_status != 'publish' ) {
+			return;
+		}
+
+		// None of our interest..
+		if ( 'nav_menu_item' === $post->post_type ) {
+			return;
+		}
+
+		$this->importing_post_types[] = $post->post_type;
+		$this->importing_post_types   = array_unique( $this->importing_post_types );
+	}
+
+	/**
+	 * After import finished, walk through imported post_types and update info.
+	 */
+	public function status_transition_bulk_finished() {
```

Comment:
I prefer grouping methods on visibility. Meaning I'd move the private method `status_transition_bulk` below this function. However, I think this is a matter of taste as the code guidelines do not mention this.